.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Tauri Docs",
+  "runArgs": [],
+  "forwardPorts": [],
+  "mounts": [],
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": {}
+  },
+  "customizations": {
+    "settings": {
+      "extensions.installRecommended": true
+    },
+    "extensions": [

```

Comment:
Not a big fan of pushing all those third party extensions in the container by default. Are they needed for usage or do they provide convenience from your personal perspective?

Reply:
The Astro one at least is basically needed for usage. Several of the others are less about being "needed" and more about providing developers with a similar setup of tools related to things like linting and task management. Technically all extensions are always optional, Notepad is the best IDE after all x)

Also it's reeeally annoying to have to manually install all recommended extensions when you load up the repo, this is the only way to make it as quick and easy to just boot up and start editing with. Remember that if it's ultimate customization you want a devcontainer may not be the best to use in the first place, and the main purpose of this devcontainer is to get people up and running quickly.

That's my take on it at least, but I'm not impossible to convince otherwise. I'm not super emotionally attached to the devcontainer.

Reply:
I'd rather trim it down to only the Astro extension. The idea of having a dev container in the first place is to get something spun up super quick and easy and we should keep the plugins to a minimum to achieve that. 

If we slim it down to only Astro's plugin then I think `extensions.installRecommended` is fine.